<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NTCNA V2X: J2735 Decoder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>6</y>
      <width>471</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>NTCNA J2735 PCAP Decoder to JSON Conversion </string>
    </property>
   </widget>
   <widget class="QLabel" name="lblInput">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>44</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Input File(s):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputFile">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>45</y>
      <width>471</width>
      <height>30</height>
     </rect>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFile">
    <property name="geometry">
     <rect>
      <x>571</x>
      <y>44</y>
      <width>84</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>File</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblOutput">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>82</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Output Dir:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outputDir">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>83</y>
      <width>471</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDir">
    <property name="geometry">
     <rect>
      <x>571</x>
      <y>82</y>
      <width>84</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Dir</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRX">
    <property name="geometry">
     <rect>
      <x>394</x>
      <y>180</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>RX</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="convertFile">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>121</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Convert File(s)</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBSM">
    <property name="geometry">
     <rect>
      <x>294</x>
      <y>206</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>BSM Counts</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMAP">
    <property name="geometry">
     <rect>
      <x>294</x>
      <y>230</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>MAP Count</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSPAT">
    <property name="geometry">
     <rect>
      <x>294</x>
      <y>250</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SPAT Count</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRTCM">
    <property name="geometry">
     <rect>
      <x>294</x>
      <y>270</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>RTCM Count</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblOther">
    <property name="geometry">
     <rect>
      <x>294</x>
      <y>290</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Other count</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblError">
    <property name="geometry">
     <rect>
      <x>294</x>
      <y>310</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Error count</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTotal">
    <property name="geometry">
     <rect>
      <x>294</x>
      <y>330</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Total Counts</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>270</y>
      <width>84</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkDebugLog">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Debug Log</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkBinMaps">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Binary MAPs</string>
    </property>
   </widget>
   <widget class="QLabel" name="msgBSMrx">
    <property name="geometry">
     <rect>
      <x>394</x>
      <y>206</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="msgMAPrx">
    <property name="geometry">
     <rect>
      <x>394</x>
      <y>230</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="msgSPATrx">
    <property name="geometry">
     <rect>
      <x>394</x>
      <y>250</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="msgRTCMrx">
    <property name="geometry">
     <rect>
      <x>394</x>
      <y>270</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="msgOtherRx">
    <property name="geometry">
     <rect>
      <x>394</x>
      <y>290</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="msgErrorCnt">
    <property name="geometry">
     <rect>
      <x>394</x>
      <y>310</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="msgTotalRx">
    <property name="geometry">
     <rect>
      <x>394</x>
      <y>330</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="msgTotalTx">
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>330</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblTX">
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>180</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>TX</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="msgBSMtx">
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>206</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="outputFile">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkConvertData">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>BSM Conversion of split files</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkSplitMaps">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Output MAPs/SPATs into file by Id</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkSeparateMaps">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Separate MAPs from SPATs</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkSplitBsms">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Split BSMs into files by Id</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>227</x>
      <y>130</y>
      <width>341</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
